Azure/Azure-AppServices-Diagnostics-Portal
# Change logic for updating detector title and description and revert logic for get detector response

## Overview


This PR is for,
1. Change logic for updating detector title and description from DetectorMetaData to ApplensGlobal and delete unnecessary service
2. Revert my change for detector-container to unsubscribe once switching to another detector, this was not working for refreshing when detector having errors

## Type of change





## How Can This Be Tested? <span>&#128269;</span>




1. When switching between detectors, check if detector name and description updated
2. Once detector having errors, click "refresh" in command bar see if refresh works

## Checklist <span>&#9989;</span>
